crates/ruff_python_formatter/src/expression/expr_dict.rs

```diff
@@ -46,7 +39,7 @@ impl FormatNodeRule<ExprDict> for FormatExprDict {
             let mut joiner = f.join_comma_separated(item.end());
 
             let mut key_value_comments = key_value_comments;
-            for (key, value) in keys.iter().zip(values) {
+            for DictItem { key, value } in items {
                 let mut key_value_pair = KeyValuePair::new(key, value);
```

Comment:
Nit. You could consider passing the item to `KeyValuePair`

Reply:
Done in https://github.com/astral-sh/ruff/pull/11267/commits/1d6bab848d0a72571e087f52502787582a44aec6